src/queryString/getQuery.js

```diff
@@ -4,7 +4,7 @@ define(function () {
      * Gets full query as string with all special chars decoded.
      */
     function getQuery(url) {
-        url = url.replace(/#.*/, ''); //removes hash (to avoid getting hash query)
+        url = url.replace(/#.*\?/, '?'); //removes hash (to avoid getting hash query)
```

Comment:
instead of replacing the RegExp you could simply delete this line if you want to allow getting the `?.+` after the `#`.

this was there because in some cases you might have URLs like: `http://example.com/foo?bar=true#lorem?ipsum=dolor`, so what should be the result of `getQuery`?